packages/payload/src/admin/components/views/Login/index.scss

```
@@ -21,4 +21,11 @@
       margin: 0;
     }
   }
+
+  &__inputWrap {
+    margin-bottom: base(0.25);
+    & > *:first-child {
```

Instead of targeting first child, it might make more sense to use `flex` with `gap`.

```suggestion
    display: 'flex';
    flex-direction: column;
    gap: base(0.25);
  }
```